normalize_repos/labels.json

```diff
@@ -0,0 +1,163 @@
+{
+  "groups": [
+    {
+      "name": "priority",
+      "labels": [
+        {
+          "name": "critical",
+          "color": "UNFAVOURABLE",
+          "description": "Issue breaks production and needs fixing asap",
+          "emoji": "\uD83D\uDFE5"
+        },
+        {
+          "name": "high",
+          "color": "NEGATIVE",
+          "description": "Issue stalls works on the project or its dependents",
+          "emoji": "\uD83D\uDFE7"
+        },
+        {
+          "name": "medium",
+          "color": "NEUTRAL",
+          "description": "Issue is not up actively harmful but that can escalate",
+          "emoji": "\uD83D\uDFE8"
+        },
+        {
+          "name": "low",
+          "color": "POSITIVE",
+          "description": "Issue doesn't hurt anyone",
+          "emoji": "\uD83D\uDFE9"
+        }
+      ]
+    },
+    {
+      "name": "status",
+      "labels": [
+        {
+          "name": "ready for dev",
+          "color": "LIGHT",
+          "description": "Issue is ready for work as blockers have been resolved",
```

Comment:
I'd remove "as blockers have been resolved", there might have not been any blockers to resolve.

Reply:
Updated in caf892e.